include/usb/class/usb_dfu.h

```diff
@@ -119,4 +119,6 @@ enum dfu_state {
 	dfuERROR,
 };
 
+void wait_for_dfu(void);
```

Comment:
I think it should be clearer, please change it to `wait_for_usb_dfu`

Reply:
Done.